test/analysis/test_summary.py

```diff
@@ -0,0 +1,31 @@
+# Run from SPRAS root with python -m test.analysis.test_summary
+
+import pandas as pd
+import os
+from glob import glob
+from src.analysis.summary.summary import summarize_networks
+
+# Notes:
+# - Column labels are required
+# - 'NODEID' is required as the first column label
+# - file_paths must be a list, even if a single file path is passed
+
+# Make output directory
+if not os.path.isdir("test/analysis/output"):
+	os.makedirs("test/analysis/output")
+
+# Toy networks
+toy_network_files = glob("test/analysis/input/toy/*")
+toy_node_table = pd.read_csv("test/analysis/input/toy_node_table.csv")
+summarize_networks(toy_network_files, toy_node_table).to_csv("test/analysis/output/toy_summary.csv", index = False)
+
```

Comment:
Instead of writing the summary table to csv, here you can compare the generated table to the stored expected table that you read from disk.  You can use `assert` when comparing the two data frames.

We'll also convert from comma to tab separated.